buildspec-unittest-python2.7.yml

```diff
@@ -0,0 +1,26 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      # install python 2.7 because it is not available on codebuild anymore
+      - yum -y groupinstall development
```

Comment:
installing them everytime might slow the CI process down? we might need to look into see if there are other images we can use.

Reply:
The test runner requires python 3, so I need to either install python 2.7 on python 3 image or install python 3 on python 2.7 image.

And it only takes 3 mins to start the build and finish everything.